postgres/lib/PostgresService.js

```diff
@@ -254,6 +254,18 @@ GROUP BY k
       return super.column_alias4(x, q)
     }
 
+    SELECT_columns(q) {
+      if (!q.SELECT.columns) return '*'
+      // REVISIT: sflight tests fail without deduplication of columns
+      // The specific column which is defined twice is "DraftAdministrativeData_DraftUUID"
```

Comment:
@patricebender @David-Kunz any ideas why for these queries the column `DraftAdministrativeData_DraftUUID` is defined double ?

Reply:
Yes, please fix this duplicate columns. 
I actually saw this deduplication code in postgres and intentionally removed it to reveal and fix the root cause instead of going workarounds.

Reply:
I have found the root cause. Here is the proposed fix https://github.com/cap-js/cds-dbs/pull/151